lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -122,6 +122,8 @@ def encoded_string(string)
         end
 
         def indent_lines(lines, failure_number)
+          return lines if failure_number.nil?
```

Comment:
This seems undesirable.  In a1936ff0332c82c1dbc0f331493750d7f5b45b77 I defined the behavior for the when `failure_number` is `nil` -- it does not print it in the output.  But I don't see a reason to also have it skip the indentation entirely.  Consider that `indentation` is one of the options to `ExceptionPresenter.new` -- if no failure number is available I don't think it makes sense to disregard the option value passed there.  Plus if you really don't want indentation, you have a way to do that (just pass `indentation: 0`).